packages/components/src/internal/components/domainproperties/DomainForm.spec.tsx

```diff
@@ -684,4 +685,117 @@ describe('DomainForm', () => {
         expect(form).toMatchSnapshot();
         form.unmount();
     });
+
+    test('using allowImportExport', () => {
+        const domain = DomainDesign.create({});
+
+        const form = mount(
+            <DomainForm
+                domain={domain}
+                onChange={jest.fn()}
+                allowImportExport={true}
+            />
+        );
+
+        expect(form.find('.domain-form-manual-section').length).toEqual(1);
+        expect(form.find('.file-form-formats').text()).toContain('.json');
+
+        form.unmount();
+    });
+
+    test('not using allowImportExport', () => {
+        const domain = DomainDesign.create({});
+
+        const form = mount(
+            <DomainForm
+                domain={domain}
+                onChange={jest.fn()}
+                allowImportExport={false}
+            />
+        );
+
+        expect(form.find('.domain-form-manual-section').length).toEqual(0);
+        expect(form.find('.file-form-formats').length).toEqual(0);
+        expect(form.find('.domain-toolbar-export-btn').length).toEqual(0);
+        expect(form.find('.domain-field-top-noBuffer').length).toEqual(2);
```

Comment:
would it make sense to add these two checks in the `using allowImportExport` test case as well?

Reply:
Done!